lib/components/ConnectModal/index.tsx

```diff
@@ -55,6 +55,7 @@ export const ConnectModal = (props: ConnectModalProps) => {
 
   const showBrave = eth?.isBraveWallet;
   const showMetamask = eth?.isMetaMask;
+  const shouldUseCoinbaseSDK = eth?.isCoinbaseWallet;
```

Comment:
naming is misleading, if isCoinbaseWallet is true it means its injected so we don't need the SDK.

Id go with this:
```suggestion
  const isInjectedCoinbase = eth?.isCoinbaseWallet;
```

Reply:
oh good point. updated.